spec.bs

```diff
@@ -1021,12 +1076,19 @@ They return a [=byte sequence=] or an error.
 1. Let |encryptedPayload| be the result of
     [[RFC9180#name-encryption-and-decryption|encrypting]] |plaintextPayload|
     with |hpkeContext| and |aad|.
+1. Return the [=byte sequence=] |encryptedPayload|.
```

Comment:
Oh hmm this algorithm does seem broken. Happy either way, but wondering if this would make more sense to do in a separate PR than as a drive-by. Separately, I think we need to handle the case that the hpke setup or encryption fails.

E.g. `If |hpkeContext| is an error, return |hpkeContext|`

Reply:
I'd punt semantic changes to another PR :)

Reply:
Sg -- let's maybe remove this line too then :)

Reply:
Done! Just pushed a fixup commit that drops the new "return" step and adds the `ignore` attribute to the now unused `encryptedPayload`. I'll make a followup PR that attempts to handle failing HPKE operations and brings back the return.